pkg/ruler/api_test.go

```diff
@@ -415,11 +419,35 @@ interval: 15s
 			err:    errors.New("invalid rules config: rule group 'rg_name' has no rules"),
 		},
 		{
-			name:   "with a a valid rules file",
+
+			name:   "with federated rules without enabled federation",
+			cfg:    defaultCfg,
+			status: 400,
+			input: `
+name: test
+interval: 15s
+source_tenants: [t1, t2]
+rules:
+- record: up_rule
+  expr: up{}
+- alert: up_alert
+  expr: sum(up{}) > 1
+  for: 30s
+  annotations:
+    test: test
+  labels:
+    test: test
+`,
+			err: errors.New("invalid rules config: rule group 'test' is a federated rule group, but rules federation is disabled; please contact your service administrator to have it enabled"),
```

Comment:
```suggestion
			err: errors.New("invalid rules configuration: rule group 'test' is a federated rule group, but rules federation is disabled; contact your service administrator to have it enabled."),
```
Are you talking about a configuration or a `config` file. (I assume the former.)

Reply:
(Not sure if the operator or end user will see this text.)

Reply:
> Are you talking about a configuration or a config file. (I assume the former.)

It's more a configuration - you're right. I'll also change the validations that include this string.


> (Not sure if the operator or end user will see this text.)

this will be seen by end users (although _this_ string in particular is part of our tests, but it matches the string that the user will see)

Reply:
also - it's a go convention to not put punctuation at the end of errors, so I'll only replace `config` with `configuration`